src/OrchardCore.Modules/OrchardCore.AuditTrail/Startup.cs

```diff
@@ -47,8 +46,8 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddScoped<INavigationProvider, AuditTrailAdminMenu>();
         services.AddScoped<INavigationProvider, AuditTrailSettingsAdminMenu>();
 
-        services.AddScoped<IDisplayDriver<ISite>, AuditTrailSettingsDisplayDriver>();
-        services.AddScoped<IDisplayDriver<ISite>, AuditTrailTrimmingSettingsDisplayDriver>();
+        services.AddSiteDisplayDriver<AuditTrailSettingsDisplayDriver>();
+        services.AddSiteDisplayDriver<AuditTrailTrimmingSettingsDisplayDriver>();
```

Comment:
```suggestion
        services.AddSiteDisplayDriver<AuditTrailSettingsDisplayDriver>()
            .AddSiteDisplayDriver<AuditTrailTrimmingSettingsDisplayDriver>();
```

Reply:
I don't want to do that as this will cause a conflict with my other PR.

Reply:
yes but we don't have to chain. Chain if needed not just because you feel you have to use it.  I usually use chaining to group thing together like to indicate that these service work together.

Reply:
Add sites drives are something related need to group, I don't think it's a big issue but seems you are worried to change your other PR :)

Reply:
but these are two separate things for two different functions. They don't depend on each other

Reply:
I'm not sure if there's something wrong with my above suggestion

Reply:
there is nothing wrong with the state before your suggestion.

Reply:
Leave my comments then